### PR TITLE
add /contingency-lists/count working on N contingency lists

### PR DESCRIPTION
**Pull request by dbraquart**

### CODE REVIEW

File: src/main/java/org/gridsuite/actions/server/ContingencyListService.java [hunk 1]
```diff
@@ -142,9 +142,25 @@ private List<Contingency> getPowsyblContingencies(PersistentContingencyList cont
         return powsyblContingencyList == null ? Collections.emptyList() : powsyblContingencyList.getContingencies(network);
     }
 
-    List<ContingencyInfos> exportContingencyList(UUID id, UUID networkUuid, String variantId) {
+    @Transactional(readOnly = true)
+    public Integer getContingencyCount(List<UUID> ids, UUID networkUuid, String variantId) {
+        Network network = getNetworkFromUuid(networkUuid, variantId);
+        return ids.stream()
+            .map(uuid -> {
+                List<ContingencyInfos> contingencyList = exportContingencyList(uuid, network);
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/actions/server/ContingencyListService.java:150]: return exportContingencyList(uuid, network).size() ?
and return List.of() from exportContingencyList

Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/actions/server/ContingencyListService.java:150]: The purpose of this TS was mainly a base optimization

Comment (dbraquart) [reply src/main/java/org/gridsuite/actions/server/ContingencyListService.java:150]: I did not want to change the existing function, was not sure about the impact.

Comment (dbraquart) [reply src/main/java/org/gridsuite/actions/server/ContingencyListService.java:150]: ok! it was not described in the ticket..

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**